packages/histoire-plugin-vue/components.d.ts

```diff
@@ -155,5 +156,6 @@ declare module '@vue/runtime-core' {
     HstTokenList: typeof HstTokenList
     HstTokenGrid: typeof HstTokenGrid
     HstCopyIcon: typeof HstCopyIcon
+    HstColorSelect: typeof HstColorSelect
```

Comment:
We also need to update the svelte types and code to add the new control.

Reply:
Can do!

Reply:
I added it to the svelte plugin and added a test to the svelte3 example